Scripts/script-ADIsUserMember.yml

```diff
@@ -3,14 +3,26 @@ commonfields:
   version: 1
 name: ADIsUserMember
 script: |-
+  res = []
+
   resp = demisto.executeCommand( 'ADGetUserGroups', demisto.args() )
   try:
       if not isError(resp[0]):
-          for d in resp[0]['Contents']:
-              if demisto.args()['groupname'] == d['name']:
-                  demisto.results("yes")
-                  sys.exit(0)
-          demisto.results("no")
+          group_names = argToList(demisto.args()['groupname'])
+          user_groups = [row['name'] for row in resp[0]['Contents']]
+          res = list(set(group_names).intersection(user_groups))
```

Comment:
Should also set into context under "ADGroupsUserIsMember" so users can pick it in playbooks. General good practice whenever possible.